README.md

```diff
@@ -24,6 +24,26 @@ env_vars:
 
 This role will update variables' value if necessary.
 
+## Pam environment variables - .pam_environment
+
+In case you want to narrow down visibility of your environment variables to only one user (for example: www-data) and not 
+expose them to everyone use env_pam_* options to set it. Read more: https://help.ubuntu.com/community/EnvironmentVariables#A.2BAH4-.2F.pam_environment
+
+```
+env_pam_path: "/var/www" # check $HOME directory for www-data user
```

Comment:
I dont think this needs to be defined. It should be enough to set username, then ansible could become that user, check home directory path and register it so it could be used to create pam_enviornment file.

Reply:
Fixed